requirements-test.txt

```diff
@@ -1,6 +1,8 @@
 -r requirements.txt
 factory-boy==2.8.1
-mypy
+isort==4.2.15
```

Comment:
Isn't isort a development dependency instead a test dependency?

Reply:
@cacarrara I really do not agree to be a test dependency either. I'm going to change.

Reply:
I moved flake8 and mypy as well. :+1: